templates/tab/js/default/public/auth-end.html

```diff
@@ -12,49 +12,40 @@
       integrity="sha384-mhp2E+BLMiZLe7rDIzj19WjgXJeI32NkPvrvvZBrMi5IvWup/1NUfS5xuYN5S3VT"
       crossorigin="anonymous"
     ></script>
+    <script type="text/javascript" src="https://alcdn.msauth.net/browser/2.21.0/js/msal-browser.min.js"></script>
     <script type="text/javascript">
+      var currentURL = new URL(window.location);
+      var clientId = currentURL.searchParams.get("clientId");
+
       microsoftTeams.initialize();
-      let hashParams = getHashParameters();
+      microsoftTeams.getContext(async function (context) {
+        const msalConfig = {
+          auth: {
+            clientId: clientId,
+            authority: `https://login.microsoftonline.com/${context.tid}`,
+            navigateToLoginRequestUrl: false
```

Comment:
do we need to set this value explicitly?

Reply:
please also help update other places if you decide to remove this config

Reply:
Yes we need to do so. Otherwise it will redirect to the auth-start page.